### ISSUE TITLE
Debug message format unification 

### ISSUE THREAD

**Issue by xmacan** [Cacti/spine#263]
Few Lines have **DEBUG: Device[xx]** and the rest **Device[44] DEBUG**

2022-05-04 10:20:03 - SPINE: Poller[1] PID[45643] PT[34376899840] DEBUG: Device[44] HT[1] In Poller, About to Start Polling
2022-05-04 10:20:03 - SPINE: Poller[1] PID[45643] PT[34376899840] Device[44] DEBUG: Entering ICMP Ping
2022-05-04 10:20:03 - SPINE: Poller[1] PID[45643] PT[34376899840] Device[44] DEBUG: Attempting to ping ......, seq 0 (Retry 0 of 3)
2022-05-04 10:20:03 - SPINE: Poller[1] PID[45643] PT[34376899840] Device[44] DEBUG: ICMP Device Alive, Try Count:1, Time:5.0600 ms
2022-05-04 10:20:03 - SPINE: Poller[1] PID[45643] PT[34376899840] Device[44] DEBUG: Entering SNMP Ping
2022-05-04 10:20:03 - SPINE: Poller[1] PID[45643] PT[34376899840] DEBUG: Device[44] HT[1] RECACHE: Processing 3 items in the auto reindex cache for '......'
2022-05-04 10:20:04 - SPINE: Poller[1] PID[45643] PT[34376899840] DEBUG: Device[44] HT[1] DEBUG: HOST COMPLETE: About to Exit Device Polling Thread Function

Spine 1.2.20

**Comment by netniV** [comment]
I would agree there are some inconsistency on the output there, likely due to bad fingers and different developers 💯

**Comment by netniV** [comment]
Don't think this made the 1.2.22 milestone though

**Comment by TheWitness** [comment]
Going to move this to 1.2.24
